### PR TITLE
[StatsPerform] Fix shot block event

### PR DESCRIPTION
**Pull request by DriesDeprest**
Fix so that no GoalkeeperEvent is created for a shot block.

image

### CODE REVIEW

File: kloppy/tests/test_opta.py [hunk 1]
```diff
@@ -469,3 +469,13 @@ def test_correct_deserialization(self, dataset: EventDataset):
         """Test if the miscontrol event is correctly deserialized"""
         event = dataset.get_event_by_id("2509132175")
         assert event.event_type == EventType.MISCONTROL
+
+
+class TestOptaBlockEvent:
+    """Tests related to deserialzing miscontrol events"""
```
Comment (JanVanHaaren) [inline kloppy/tests/test_opta.py:475]: "deserialzing" -> "deserializing"

File: kloppy/tests/test_opta.py [hunk 2]
```diff
@@ -469,3 +469,13 @@ def test_correct_deserialization(self, dataset: EventDataset):
...
+
+    def test_correct_deserialization(self, dataset: EventDataset):
+        """Test if the miscontrol event is correctly deserialized"""
```
Comment (JanVanHaaren) [inline kloppy/tests/test_opta.py:478]: The documentation needs to be updated.

**Comment by DriesDeprest** [comment]
@JanVanHaaren done

**Review by JanVanHaaren — APPROVED** [review]
This pull request looks good to me. Thank you!

**Pull request merged**